tedana/tests/test_stats.py

```diff
@@ -1,13 +1,80 @@
 """
-Tests for tedana.stats.get_coeffs
+Tests for the tedana stats module
 """
-
 import numpy as np
 import pytest
+import random
 
+from tedana.stats import computefeats2
 from tedana.stats import get_coeffs
+from tedana.stats import getfbounds
+
+
+"""
+Tests for computefeats2
+"""
+def test_break_computefeats2():
+    """
+    Ensure that computefeats2 fails when input data do not have the right
+    shapes.
+    """
+    n_samples, n_vols, n_comps = 10000, 100, 50
+    data = np.empty((n_samples, n_vols))
+    mmix = np.empty((n_vols, n_comps))
+    mask = np.empty((n_samples))
 
+    data = np.empty((n_samples))
+    with pytest.raises(ValueError) as e_info:
```

Comment:
why not just have this as a straight assertion ?

Reply:
Actually, `pytest.raise` would make more sense: http://doc.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions